soc/xtensa/intel_adsp/ace/include/intel_ace15_mtpm/adsp_shim.h

```diff
@@ -140,7 +140,7 @@ struct ace_dfpmccu {
 
 #endif /* _ASMLANGUAGE */
 
-#define ACE_CLKCTL_WOVCRO    BIT(4)	  /* Request WOVCRO clock */
+#define ACE_CLKCTL_WOVCRO    BIT(21)	  /* Request WOVCRO clock */
```

Comment:
Can you expand a a bit why this is a safe change to do? Was the code doing incorrect thing before (i.e. this is a fix), or was this in practise unused so this correction has no functional difference?

Reply:
This is a fix. don't know where the value 0x10 comes from, but is CLKCTL bit 4 is used for something other than requesting WOVCRO (at least for ACE).

Maybe it's worth asking if anyone knows why we do this exercise with WOVCRO during init?

```c
ACE_DfPMCCU.dfclkctl |= ACE_CLKCTL_WOVCRO;
if (ACE_DfPMCCU.dfclkctl & ACE_CLKCTL_WOVCRO) {
	ACE_DfPMCCU.dfclkctl = ACE_DfPMCCU.dfclkctl & ~ACE_CLKCTL_WOVCRO;
} else {
	platform_lowest_freq_idx = ADSP_CPU_CLOCK_FREQ_LPRO;
}
```